engine/kafka/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaUtils.scala

```diff
@@ -32,6 +31,13 @@ object KafkaUtils extends LazyLogging {
     props.setProperty("client.id", sanitizeClientId(id))
   }
 
+  def validateTopicsExistence(topics: List[PreparedKafkaTopic], kafkaConfig: KafkaConfig): Unit = {
+    new CachedTopicsExistenceValidator(
+      kafkaConfig = kafkaConfig,
+      config = CachedTopicsExistenceValidatorConfig.DefaultConfig)
```

Comment:
Can we decode validator config from kafkaConfig? or some other config? Currently it's quite cumbersome to override config without coding?

Reply:
I've added field for that in KafkaConfig